### ISSUE TITLE
Varnish 5 Support

### ISSUE THREAD

**Issue by MikeParkin** [PHOENIX-MEDIA/Magento-PageCache-powered-by-Varnish#61]
Hello,

Does this module have Varnish 5.x support?

Cheers
Mike

**Comment by PHOENIX-MEDIA** [comment]
No, at least not officially. But as far as I know Varnish5 requires only minimum adjustments to the VCL. Checking https://varnish-cache.org/docs/5.0/whats-new/upgrading-5.0.html I don't see obvious incompatibilities. The module itself won't need any adjustments.

If you want to give it a try I would be happy accept your PR.
